src/get_maintainer_wrapper.sh

```diff
@@ -1,90 +1,240 @@
 . $src_script_path/kwio.sh --source-only
 
-
+# Prints the authors of a given file or files inside a given dir.
+#
+# @parameter The argument is a file or directory path
 function print_files_authors()
 {
-    # TODO: currently only authors found in a single line MODULE_AUTHOR
-    # statement are captured and printed. In the future, it would be
-    # nice to look for multiline MODULE_AUTHOR statatements such as:
-    #
-    # MODULE_AUTHOR ("a_long_email@xx.com" \
-    #                "another_long@yy.com" )
-    # and:
-    #
-    # MODULE_AUTHOR ("a_long_email@xx.com"
-    #                "another_long@yy.com" )
-    #
-    # which are, both, valid C statements. In the currently
-    # implementation neither of these four emails will be printed.
-
-    local FILE_OR_DIR=$1
-    local files=( )
-    if [[ -d $FILE_OR_DIR ]]; then
-        for file in $FILE_OR_DIR/*; do
-            if [[ -f $file ]]; then
-                files+=($file)
-            fi
-        done
-    elif [[ -f $FILE_OR_DIR ]]; then
-        files+=($FILE_OR_DIR)
+  # TODO: currently only authors found in a single line MODULE_AUTHOR
+  # statement are captured and printed. In the future, it would be
+  # nice to look for multiline MODULE_AUTHOR statatements such as:
+  #
+  # MODULE_AUTHOR ("a_long_email@xx.com" \
+  #                "another_long@yy.com" )
+  # and:
+  #
+  # MODULE_AUTHOR ("a_long_email@xx.com"
+  #                "another_long@yy.com" )
+  #
+  # which are, both, valid C statements. In the currently
+  # implementation neither of these four emails will be printed.
+
+  local FILE_OR_DIR=$1
+  local files=( )
+  if [[ -d $FILE_OR_DIR ]]; then
+    for file in $FILE_OR_DIR/*; do
+      if [[ -f $file ]]; then
+        files+=($file)
+      fi
+    done
+  elif [[ -f $FILE_OR_DIR ]]; then
+    files+=($FILE_OR_DIR)
```

Comment:
How about adding an else operation for the case that something weird is passed in $FILE_OR_DIR? Another question, how about check if the user has permission to the directory and file?
IMHO this can be good to provide precise information for users.